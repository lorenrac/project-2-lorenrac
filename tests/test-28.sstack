VAR sentence = "This sentence should be printed one word per line."
WHILE (sentence ? " ") {
  PRINT sentence / " "
  sentence = sentence % " "
}
PRINT sentence
