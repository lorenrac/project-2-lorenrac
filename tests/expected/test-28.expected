This
sentence
should
be
printed
one
word
per
line.
